db/schema.rb

```diff
@@ -36,7 +36,7 @@
     t.boolean  "color"
     t.integer  "game_id"
     t.integer  "user_id"
-    t.boolean  "captured",   default: false
```

Comment:
you need to drop your tables and rebuild your databases for this to be fixed  (rake db:drop db:create db:migrate)